app/models/data_request.rb

```diff
@@ -4,15 +4,30 @@ class DataRequest < ApplicationRecord
   has_many   :data_request_logs, after_add: :update_cached_attributes
 
   validates :location, presence: true, length: { maximum: 500 }
-  validates :data, presence: true
+  validates :request_type, presence: true
   validates :offender_sar_case, presence: true
   validates :user, presence: true
   validates :date_requested, presence: true
   validates :cached_num_pages, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validate :validate_request_type_note
+  validate :validate_from_date_before_to_date
 
   before_validation :set_date_requested
   before_validation :clean_attributes
 
+
+  enum request_type: {
+    offender: 'offender',
+    all_prison_records: 'all_prison_records',
+    all_nomis_records: 'all_nomis_records',
+    nomis_contact_logs: 'nomis_contact_logs',
+    probation_records: 'probation_records',
+    prison_and_probation_records: 'prison_and_probation_records',
+    other: 'other'
+  }
+
```

Comment:
looked at the ticket,  there is no type called "offender",  do you need to remove it?

Reply:
hmm, just checked the rest of codes, seems you use "offender" for so many places..... but I didn't see it from Jira ticket

Reply:
Oh man - Thanks! I inherited that from somewhere else and hadn't checked properly, just saw there were 7 🤦 

I used it EVERYWHERE 😂 - changing